api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala

```diff
@@ -145,10 +145,11 @@ class ApiWorksTest
   }
 
   it(
-    "should return the requested page of results when requested with page & pageSize") {
```

Comment:
Squeezed the links tests in here. I think it makes sense as this test should really be covering the edge cases (first/last) too.